Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestPage.tsx

```diff
@@ -130,11 +124,7 @@ const ContestPage = () => {
                 )
                 : isParticipationOfficial && contest?.isOnline && !isUserAdmin && !isUserParticipant
                     ? (
-                        <ContestModal
-                          contest={modalContest}
-                          isShowing={isShowing}
-                          toggle={toggle}
-                        />
+                        <ContestModal contest={modalContest} />
```

Comment:
Parenthesis are not need around the modal component

Reply:
Fixed.